sonaranalyzer-dotnet/its/regression-test.ps1

```diff
@@ -47,9 +47,9 @@ function Build-Project([string]$ProjectName, [string]$SolutionRelativePath) {
 function Initialize-ActualFolder() {
     $methodTimer = [system.diagnostics.stopwatch]::StartNew()
 
-    Write-Host "Initializing the actual issues folder with the expected  result"
+    Write-Host "Initializing the actual issues folder with the expected result"
```

Comment:
Unrelated: increasing the visibility of some the messages. For some reason, the folders on my machine aren't deleted first time the script is run.